IPython/external/qt_loaders.py

```diff
@@ -179,8 +179,12 @@ def import_pyqt4(version=2):
     import sip
 
     if version is not None:
+        sip.setapi('QDate', version)
+        sip.setapi('QDateTime', version)
         sip.setapi('QString', version)
-        sip.setapi('QVariant', version)
```

Comment:
What happened to QVariant? Is that no longer needed?

Also, you'll need to update the docstring of this function.